lib/alexa_request_verifier/cert_cache.ex

```diff
@@ -0,0 +1,79 @@
+defmodule AlexaRequestVerifier.CertCache do
+  @moduledoc """
+  This module caches successfully verified Amazon certificates.
+  """
+
+  use GenServer
+
+  require Logger
+
+  @server __MODULE__
+  @table :alexa_request_verifier_certs
+
+  # client
+
+  @doc """
+  Return either the last successfully verified Amazon certificate,
+  or nil if this is the first time the cache has been accessed.
+  """
+  @spec get_cert(cert_url :: String.t()) :: String.t() | nil
+  def get_cert(cert_url) do
+    GenServer.call(@server, {:fetch, cert_url})
```

Comment:
This is not a good cache interface because you are serializing all validation requests on your cache server process. It would be better to use e2qc or protected ets for this. Take a look at the assistant loader in core for an example. You probably don't need eviction, so you can ignore the `:hit` message.

It is also better to model cache misses as a single transaction instead of separate get/store messages.